app/assets/javascripts/admin/controllers/admin-email-index.js.es6

```diff
@@ -34,8 +34,8 @@ export default Ember.Controller.extend({
         data: { email_address: this.get("testEmailAddress") }
       })
         .then(
-          function() {
-            self.set("sentTestEmail", true);
+          function(response) {
+            self.set("sentTestEmailMessage", response.sentTestEmailMessage);
```

Comment:
While you're in there, you could improve this to use the `(response) =>` form of functions, which would mean you could use `this` instead of `self`. I'd like to remove the `self` pattern from our code.